src/ui.rs

```diff
@@ -223,10 +226,13 @@ fn make_steppy_knob(
 
         Knob::custom(
             cx,
-            UiData::params.get(cx).get_parameter_default(param_index),
-            UiData::params.map(move |params| {
-                params.get_parameter(param_index)
-            }),
+            UiData::params
+                .get(cx)
+                .get_parameter_by_index(param_index)
```

Comment:
This also gets a bit less wordy if you implement the `Index<usize>` trait.

```rust
UiData::params.get(cx)[param_index as usize]
```
I personally like that more, what do you think?